packages/app-config/scripts/package-subtree-local.sh

```
@@ -0,0 +1,1201 @@
+  local message="$2"
+  local is_user_error=${3:-false}
+
+  if [ -n "$message" ]; then
+    error_msg "$message"
+  fi
+
+  if [ "$is_user_error" = "true" ] && [ -n "${npm_config_user_agent:-}" ]; then
+    warning_msg "Exiting with code 0 for npm compatibility (original exit code: $exit_code)"
+  fi
+
+  exit $exit_code
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)